sb_vision/tokens.py

```diff
@@ -222,7 +219,10 @@ def infer_location_from_homography_matrix(
         )
 
         # Polar co-ordinates in the 3D world, relative to the camera
-        self.polar = self.cartesian_to_polar(self.cartesian)
+        self.polar = cartesian_to_legacy_polar(self.cartesian)
```

Comment:
what's the point of both `polar` and `legacy_polar`? Surely this is unnecessary?

Reply:
I want to remove `polar` soon so that internally we only have `legacy_polar`. By having both for a bit we get to so that stepwise, rather than needing to update everything all in one go.